pkg/project/testdata/project.yaml

```diff
@@ -0,0 +1,3 @@
+project:
```

Comment:
is this inspired by the `project.yaml` from SO? 

What's the file for? Let's document it, somehow

Reply:
Added a few comments